pkg/setup/clean.go

```diff
@@ -127,6 +127,7 @@ func (e *Environment) Clean() error {
 			e.cleanMounts()
 		}
 		toRemove = append(toRemove, e.kubeletRootDir)
+		toRemove = append(toRemove, "/var/log/pods")
```

Comment:
Use `pkg.run.KubeletCRILogs` instead of hardcoded path? (needs to be public)

Reply:
The path is non configurable anyway. 
But I can refactor this to use the same constant 👍